<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pingset</class>
 <widget class="QWidget" name="Pingset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="pingtextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pingpushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>ping</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="pingtextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ip:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>負載：可填可不填,第一塊爲IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>第二塊爲ICMP的.</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="IPloadtextEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ICMPtextEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>由于程序问题，请点击右侧查看icmp应答，才能解析</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
